src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.mvc.pathmatch.matching-strategy=ant-path-matcher
```

Comment:
@pvannierop @oplantalech actually one comment about this. Is there a way we can pass this in another way. Reason I'm asking is that I would ideally prefer installers having to update their existing `application.properties` files

Reply:
@inodb
How this prop is passed to the application indeed is the responsibility of the installer. Or did not mean _I would ideally prefer installers **not** having to update their existing application.properties files_?

Reply:
@inodb 
If the latter, we can try an approach with multiple property sources like described [here](https://www.baeldung.com/properties-with-spring#1-defining-multiple-property-locations).

Reply:
Sorry yes I meant **not** having to update their existing files. Especially like in this case where it really shouldn't be a user customizable property

Spring often has ways of configuring this through either properties or having it hardcoded. I prefer the latter in this case, since it shouldn't be configurable. Multiple property files could work too otherwise by putting all "don't touch this" configurable spring stuff in one file